opm/material/fluidsystems/BlackOilFluidSystem.hpp

```diff
@@ -657,6 +691,26 @@ class BlackOil : public BaseFluidSystem<Scalar, BlackOil<Scalar> >
         referenceDensity_.resize(numRegions);
     }
 
+    template <class LhsEval, class FluidState>
+    static LhsEval getRs_(const FluidState& fluidState, unsigned regionIdx)
+    {
+        typedef Opm::MathToolbox<typename FluidState::Scalar> FsToolbox;
+
+        const auto& XoG =
+            FsToolbox::template toLhs<LhsEval>(fluidState.massFraction(oilPhaseIdx, gasCompIdx));
+        return XoGToRs(XoG, regionIdx);
```

Comment:
The method name XoGToRs() is unfortunate, it looks like a class (capital initial).